modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -9539,6 +9539,71 @@ public void testPostCustomObjectEntryWithStatus() throws Exception {
 				_objectDefinition1.getRESTContextPath(), Http.Method.POST));
 	}
 
+	@Test
+	public void testPostObjectEntryExpire() throws Exception {
+
+		// Company scoped object entry
+
+		ObjectEntry companyScopedObjectEntry =
+			_objectEntryLocalService.addObjectEntry(
+				0, TestPropsValues.getUserId(),
+				_objectDefinition1.getObjectDefinitionId(),
+				ObjectEntryFolderConstants.
+					PARENT_OBJECT_ENTRY_FOLDER_ID_DEFAULT,
+				null,
+				HashMapBuilder.<String, Serializable>put(
+					_OBJECT_FIELD_NAME_TEXT, RandomTestUtil.randomString()
+				).build(),
+				ServiceContextTestUtil.getServiceContext());
```

Comment:
Can you add an `Assert` after adding an entry asserting that the code was published?
Before expiring the entry.